src/components/site/PageHeader.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PageHeader from './PageHeader';
+
+it('renders without crashing', () => {
```

Comment:
@sgrg1 looks like this file still needs the updated formatting.

Reply:
done